### ISSUE TITLE
Looking to use this and need some info

### ISSUE THREAD

**Issue by Fyb3roptik** [lunny/axmlParser#4]
Any chance you can update the usage instructions? I would like to be able to get the package name and version info from an uploaded apk file. Thanks!

**Comment by lunny** [comment]
I think you can see the README about how to use it.

**Comment by Fyb3roptik** [comment]
listener := new(AppNameListener)
_, err := ParseApk(apkfilepath, listener)

Here is the README code, but the _ wont have anything assigned to it. How do I get the info? Is it attached to the listener?

**Comment by lunny** [comment]
@Fyb3roptik yes, all infos will be attached to Listener. AppNameListener will only get app name. You can implement your own Listener to get what you want from the xml file.
